tests/test_dataclasses.py

```diff
@@ -1851,6 +1853,29 @@ def test_clone(self):
             assert round(abs(c.sites[1].position[i] - 1.0), 7) == 0
 
 
+@skip_atomistic
+class TestAtomisticStructureData:
```

Comment:
Only has one test, so I think there is no needed to use class to categorize tests.

Reply:
I agree, we can make a class if in the future we need more tests (I doubt it as everything will go into aiida-atomistic)